lib/exitCodes.ts

```diff
@@ -1,28 +1,79 @@
 import {Logger} from './logger2';
 
 const CONFIG_ERROR_CODE = 105;
+const BROWSER_ERROR_CODE = 135;
```

Comment:
I think `BROWSER_ERROR` is too ambiguous - let's make this `BROWSER_CONNECT_ERROR_CODE`. It's long but our users shouldn't need to type it.